__tests__/app.test.js

```diff
@@ -71,6 +71,106 @@ describe("GET /api/reviews", () => {
 				});
 			});
 	});
+
+	test("status:200, category query, responds with a filtered reviews array", () => {
+		return request(app)
+			.get("/api/reviews?category=dexterity")
+			.expect(200)
+			.then(({ body }) => {
+				const { reviews } = body;
+				expect(reviews).toHaveLength(1);
+				reviews.forEach((review) => {
+					expect.objectContaining({
+						owner: expect.any(String),
+						title: expect.any(String),
+						designer: expect.any(String),
+						review_id: expect.any(Number),
+						category: "dexterity",
+						review_img_url: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						comment_count: expect.any(String)
+					});
+				});
+			});
+	});
+
+	test("status:200, sort_by query, responds with a sorted reviews array", () => {
+		return request(app)
+			.get("/api/reviews?sort_by=votes")
+			.expect(200)
+			.then(({ body }) => {
+				const { reviews } = body;
+				expect(reviews).toHaveLength(13);
+				reviews.reduce((pre, cur) => {
+					if (pre && cur) {

```

Comment:
We are not doing anything with the output of reduce, and pre and cur here are two objects so we could remove the reduce 🙂